yarn/datadog_checks/yarn/yarn.py

```diff
@@ -234,13 +235,15 @@ def _yarn_app_metrics(self, rm_address, app_tags, addl_tags):
         """
         Get metrics for running applications
         """
+        collect_apps_all_states = self.instance.get('collect_apps_all_states', False)
```

Comment:
Would that makes sense to also add the option to the global config and not only per instance?

Reply:
Yes, definitely! I'll add that